docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -14,6 +14,7 @@ Tag <|-- Interest
 UniquePersonList -right-> Person
 
 Person -up-> "*" Tag
+Tag -> "*" Interest
```

Comment:
This should be either navigability or 1-to-1 relationship though, its not that tag has many none/many interest